instance/tests/models/test_openedx_storage_mixins.py

```diff
@@ -380,15 +386,18 @@ def test_deprovision_s3_delete_bucket_fails(self, s3_connection, connect_iam):
         instance.s3_access_key = 'test'
         instance.s3_secret_access_key = 'test'
         instance.s3_bucket_name = 'test'
+        instance.s3_region = 'test'
         with self.assertLogs("instance.models.instance"):
             instance.deprovision_s3()
         # Since it failed deleting the bucket, s3_bucket_name should not be empty
         self.assertEqual(instance.s3_bucket_name, "test")
+        self.assertEqual(instance.s3_region, "test")
```

Comment:
@lgp171188 You could add another comment above this line that says something like:

```python
# We always want to preserve information about a client's preferred region, so s3_region should not be empty.
```